### PR TITLE
BZ2024198 incorrect STS secret format

### PR DESCRIPTION
**Pull request by jeana-redhat**
For https://bugzilla.redhat.com/show_bug.cgi?id=2024198

Preview: https://deploy-preview-38880--osdocs.netlify.app/openshift-enterprise/latest/authentication/managing_cloud_provider_credentials/cco-mode-sts

### CODE REVIEW

**Comment by jeana-redhat** [comment]
@joelddiaz thx for filing the BZ for this! I think this applies to 4.7+ unless the secret format has changed between 4.7 and now, please lmk and take a look to verify the fix in any case.

**Comment by joelddiaz** [comment]
Format hasn't changed since the original Tech Preview for 4.7

File: authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc [hunk 1]
```diff
@@ -45,8 +45,10 @@ metadata:
   namespace: <target-namespace> <1>
   name: <target-secret-name> <2>
 data:
-  role_name: <operator-role-name> <3>
-  web_identity_token_file: <path-to-token> <4>
+  credentials: |
```
Comment (joelddiaz) [inline authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc:48]: for more clarity i would change it from `data` to `stringData` and from
`credentials: |` to
`credentials: |-`
As show in https://github.com/openshift/cloud-credential-operator/blob/master/pkg/cmd/provisioning/aws/create-iam-roles.go#L28

Comment (jeana-redhat) [reply authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc:48]: ah ok - will do!

**Comment by jeana-redhat** [comment]
@lwan-wanglin PTAL when you can :)

File: authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc [hunk 2]
```diff
@@ -45,8 +45,10 @@ metadata:
   namespace: <target-namespace> <1>
   name: <target-secret-name> <2>
 data:
```
Comment (lwan-wanglin) [inline authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc:47]: @jeana-redhat I read @joelddiaz 's review comment, the `data` would be `stringData` .
data field will show a base64 encode string while stringData shows a readable string.

Comment (jeana-redhat) [reply authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc:47]: Thanks, I apparently could not read. Will fix that and get it updated :+1:

**Comment by bmcelvee** [comment]
LGTM!

**Comment by lwan-wanglin** [comment]
LGTM!

**Pull request merged**